### PR TITLE
cli: implement attach

### PR DESCRIPTION
**Pull request by lucab**
This implements the `attach` subcomand.

### CODE REVIEW

File: internal/common/common_test.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+// limitations under the License.
+
+package common
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestLookupBlockdev(t *testing.T) {
```
Comment (cgonyeo) [inline internal/common/common_test.go]: Is it too unreliable to test a positive case both here and in `TestLookupVolName`? Is it safe to assume that `/dev/loop0` is available?

**Comment by lucab** [comment]
@dgonyeo I added two positive tests, conditional on being able to `modprobe loop`. Both travis and my local machine seems to be fine with that.

**Review by cgonyeo — APPROVED** [review]

**Comment by cgonyeo** [comment]
Both this and #2 look fine to me. I'll review #4 on Monday (taking PTO tomorrow).

**Comment by lucab** [comment]
@dgonyeo thanks, I've rebased this only to get rid of the duplicate commit from #2. Merging.

**Pull request merged**